lab_1/scripts/build-client.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+echo "Enter source code location"
+read SOURCE_LOCATION
+cd $SOURCE_LOCATION
+npm install
+declare -A env_variable_names
+env_variable_names[development]=""
+env_variable_names[production]="production"
+echo "Enter environment"
+echo "1 - development"
+echo "2 - production"
+while true
```

Comment:
there is "select"command in bash for such cases - https://linuxize.com/post/bash-select/